x-pack/plugins/reporting/server/lib/screenshots/open_url.ts

```diff
@@ -18,14 +19,15 @@ export const openUrl = async (
   url: string,
   pageLoadSelector: string,
```

Comment:
Would you mind doing a variable rename of `pageLoadSelector` to `waitForSelector`? It'll let there be more shorthand syntax on line 30